test/common/orca/orca_parser_test.cc

```diff
@@ -0,0 +1,72 @@
+#include <string>
+
+#include "source/common/common/base64.h"
+#include "source/common/orca/orca_parser.h"
+
+#include "test/test_common/status_utility.h"
+#include "test/test_common/utility.h"
+
+#include "absl/status/status.h"
+#include "xds/data/orca/v3/orca_load_report.pb.h"
+
+namespace Envoy {
+namespace Orca {
+namespace {
+
+// Returns an example OrcaLoadReport proto with all fields populated.
+static xds::data::orca::v3::OrcaLoadReport ExampleOrcaLoadReport() {
```

Comment:
style: `exampleOrcaLoadReport`

Reply:
@blake-snyder please update the function name.